applications/DEMApplication/python_scripts/analytic_tools/analytic_data_procedures.py

```diff
@@ -71,12 +70,29 @@ def UpdateVariables(self, n_particles_old, n_mass_old):
     def ClearData(self):
         self.face_watcher.ClearData()
 
+
+class ParticleAnalyzerClass:
+    def __init__(self, model_part):
+        self.analytic_model_part = model_part
+        self.particle_analyzer = AnalyticParticleWatcher()
+
+    def MakeAnalyticsMeasurements(self):
+       self.particle_analyzer.MakeMeasurements(self.analytic_model_part)
+
+    def SetNodalMaxImpactVelocities(self):
+        self.particle_analyzer.SetNodalMaxImpactVelocities(self.analytic_model_part)
+
+    def SetNodalMaxFaceImpactVelocities(self):
+        self.particle_analyzer.SetNodalMaxFaceImpactVelocities(self.analytic_model_part)
+
+
 class FaceAnalyzerClass:
```

Comment:
FacesAnalyzerClass, or SurfacesAnalyzerClass, or FacesAnalyzer, or SurfacesAnalyzer (we know it is a Class)

Reply:
SurfacesAnalyzerClass sounds good to me.
@jgonzalezusua @GuillermoCasas